packages/web/src/pages/chat-page/components/CreateChatModal.tsx

```diff
@@ -48,10 +50,14 @@ export const CreateChatModal = () => {
 
   const loadMore = useCallback(() => {
     if (currentUser) {
-      dispatch(followersUserListActions.setFollowers(currentUser?.user_id))
-      dispatch(userListActions.loadMore(FOLLOWERS_USER_LIST_TAG))
+      if (defaultUserList === 'chats') {
+        dispatch(fetchMoreChats())
```

Comment:
nit: feels like there's potentially a cleaner way to do this, but i think this is fine since we're probably not gonna extend this functionality beyond these two cases.

Reply:
Yeah you're right, there's likely a cleaner way. But yeah we can cross that bridge later